ui/perfherder/RevisionInformation.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { react2angular } from 'react2angular/index.es2015';
+
+import perf from '../js/perf';
+
+import { getRevisionUrl } from './helpers';
+
+export default class RevisionInformation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      originalProject,
+      originalRevision,
+      newProject,
+      newRevision,
+      originalResultSet,
+      newResultSet,
+      selectedTimeRange,
+    } = this.props;
+
+    return (
+      <ul className="list-inline push-information">
+        {originalRevision && (
+          <li className="list-inline-item">
+            <strong>Base</strong> -&nbsp;
```

Comment:
Since the content of each list item is just a formatted string, I wonder if there's an easy way to use a template literal or formatting function.